schema.sql

```diff
@@ -58,4 +58,18 @@ create table visits (
 	vets_id int references vets(id),
 	visit_date date,
 	primary key (id)
-);
\ No newline at end of file
+);
+
+-- Performance and indexing.
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animals_id_desc ON owners(animals_id desc);
+
+
+CREATE INDEX email_asc ON owners(email ASC);
+
+
+CREATE INDEX vets_id_desc ON visits(vets_id asc);
+
```

Comment:
- [ Optional]This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project.

As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:

` CREATE INDEX ON owners ((lower(email)));`

Reply:
Ok. thanks